app/kubemci/cmd/create.go

```diff
@@ -87,6 +87,12 @@ type CreateOptions struct {
 	// the resource does not exist, or is already the correct
 	// value, then 'force' is a no-op.
 	ForceUpdate bool
+	// Name of the namespace for the ingress. Overrides the namespace in the IngressFilename (if present).
```

Comment:
This is not true anymore? (it does not override, but generates an error if there is a mismatch)
(same for static-ip)